utilities/xml_to_csv_parser.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/local/bin/python3
```

Comment:
shebang is wrong there
What about using an entrypoint in `setup.py` instead so that setuptools will generate the entry point with the right shebang for us?

See how ros2cli is working: https://github.com/ros2/ros2cli/blob/master/ros2cli/setup.py